tools/dds/dds-config/readme.md

```diff
@@ -0,0 +1,61 @@
+# dds-config tool
+
+## Goal
+This tool is used to configer DDS based device and ensure it operates correctly within a DDS environment.  
+It allows users to manage connection priorities, enabling the selection of either DDS or USB as the preferred connection method when both options are available.
+## Description
+The dds-config tool allows users to prioritize either the DDS or USB connection and configure network parameters for the device.  
+It also provides information about the connected devices when executed.
+
+## Command Line Parameters
+| Flag | Description |
+|---|---|
+|-h --help|Show command line help menu|
+|--usb-first|Prioritize USB before Ethernet|
```

Comment:
Will it fallback to Eth?

Reply:
Yes, it will, can add the remark here as in `eth-first` flag

Reply:
added fallback remark